docs/UserGuide.md

```diff
@@ -3,11 +3,39 @@ layout: page
 title: User Guide
 ---
 
-FriendDex is a **relationship management tool for CLI enthusiasts** looking to enhance their social life while not compromising on getting things done quickly. Managing your relationship goals should not be any more tedious than coding.
+Welcome to **FriendDex**, a **relationship management tool** looking to enhance your social life while not compromising on getting things done quickly. 
+Managing your relationships should not be any more tedious than doing your laundry.
+
+Although optimized for Command Line Interface enthusiasts, it should be easy enough for anyone to use it.
+To help you get started, read the [Quick start](#quick-start) section below.
 
 * Table of Contents
 {:toc}
 
+--------------------------------------------------------------------------------------------------------------------
+## About 
+This document shows you the details of the [User Interface](#UI) [Features](#Features) present in **FriendDex**. 
```

Comment:
Copy nit: "This document walks you through the User Interface and Features present in FriendDex".